proto/v1/project_service.proto

```diff
@@ -267,4 +297,111 @@ enum ProjectRole {
   PROJECT_ROLE_UNSPECIFIED = 0;
   PROJECT_ROLE_OWNER = 1;
   PROJECT_ROLE_DEVELOPER = 2;
-}
\ No newline at end of file
+}
+
+message GetProjectIssueRequest {
+  // The name of the issue to retrieve.
+  // Format: projects/{project}/issues/{issue}
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
So we require issue name to be unique?

Reply:
The previous issue name becomes the title. This name is the unique resource URI of an issue.